jsk_topic_tools/src/connection_based_nodelet.cpp

```diff
@@ -107,19 +108,20 @@ namespace jsk_topic_tools
     }
   }
 
-#if nodelet_VERSION_MINOR > 9 || (nodelet_VERSION_MINOR == 9 && nodelet_VERSION_PATCH >= 11)
   bool ConnectionBasedNodelet::warnNoRemap(const std::vector<std::string> names)
   {
     bool no_warning = true;
     // standalone
     ros::M_string remappings = ros::names::getRemappings();
+#if NODELET_VERSION_MINIMUM(1, 9, 11)
```

Comment:
Why not capture the whole block of this function?
`remappings` will be an empty vector, and all names will be warned if nodelet version is old?

Reply:
I change like this because `log_utils` works like this https://github.com/jsk-ros-pkg/jsk_common/blob/master/jsk_topic_tools/src/log_utils.cpp#L44-L60 .
If i remove this section for `nodelet==1.9.10`, this function works as same as `log_utils warnNoRemap`. (but I'm not sure, so I ask you to review this PR).
Amn't I correct?

Reply:
`log_utils::warnNoRemap` is supposed to be used only with a ROS node (not nodelet). As `ConnectionBasedNodelet` is a nodelet, we cannot use the same logics.

Reply:
I see.
So what is you recommendation?
if `nodelet==1.9.10` is used, it is better to return just `no_warning` `true` in `ConnectionBasedNodelet::warnNoRemap`?

Reply:
yeah.

Reply:
ok, I see.
I will update this PR.

Reply:
I updated this PR to follow your review.